mobile/lib/domain/services/sync_stream.service.dart

```
@@ -59,11 +57,9 @@ class SyncStreamService {
 
   Future<void> _handleSyncData(
     SyncEntityType type,
-    // ignore: avoid-dynamic
```

Why are these removed? We extend the recommended rules of dcm and this one is part of it. Removing it should show us a violation for the below dynamic usage

---

I found out that apparently, DCM silently ignores if you use a preset that doesn't exist, and my installation for some reason doesn't include the "recommended" preset. I'll see if I can fix that and then check again.

---

> until the corresponding rules are actually enabled, or we need to enable the rules and fix or ignore all violations.

It was probably throwing you the warning due to the licensing issue. I'll mark this as resolved since we are to remove all the rules for now.